core/src/main/scala/org/apache/spark/deploy/rest/StandaloneRestServer.scala

```diff
@@ -174,6 +174,13 @@ private[rest] class StandaloneSubmitRequestServlet(
     conf: SparkConf)
   extends SubmitRequestServlet {
 
+  val envVariablePattern = "\\{\\{[A-Z_]+\\}\\}".r
```

Comment:
Hmm, where is this `envVariablePattern` used?

Reply:
Oh, right. Now, it's no op. Let me clean up it.
I used it for Scala 2.12 patch.